### DISCUSSION TITLE
Telegraf missing after several fresh installs

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#6847] [Unsupported Versions]
Hi, I've tried two different computers with brand new, distributed sensor installations on Ubuntu 18.04 and telegraf is always listed as missing when I run so-status and the SOC shows a fault. I did a fresh install of Ubuntu and SO on both computers. No firewalls have changed from when it was working to when it wasn't. There is nothing in the error.log. I've tried `salt-call state.highstate`. I don't see anything about telegraf in the manual. The last computer I used was one that had been working fine when it was running 2.3.90. Now, on 2.3.91, I get this fault.

This is what the sosetup.log says: 

Line 1596: +log_level: error
	Line 1597: +log_level_logfile: error
	Line 1833:                   +log_level: error
	Line 1834:                   +log_level_logfile: error
	Line 7512: [ERROR   ] Failed to pull somanager:5000/security-onion-solutions/so-telegraf:2.3.91: Error 404: manifest for somanager:5000/security-onion-solutions/so-telegraf:2.3.91 not found: manifest unknown: manifest unknown
	Line 7512: [ERROR   ] Failed to pull somanager:5000/security-onion-solutions/so-telegraf:2.3.91: Error 404: manifest for somanager:5000/security-onion-solutions/so-telegraf:2.3.91 not found: manifest unknown: manifest unknown
	Line 7516: [ERROR   ] Failed to pull somanager:5000/security-onion-solutions/so-telegraf:2.3.91: Error 404: manifest for somanager:5000/security-onion-solutions/so-telegraf:2.3.91 not found: manifest unknown: manifest unknown
	Line 7516: [ERROR   ] Failed to pull somanager:5000/security-onion-solutions/so-telegraf:2.3.91: Error 404: manifest for somanager:5000/security-onion-solutions/so-telegraf:2.3.91 not found: manifest unknown: manifest unknown
	Line 9925:      Comment: Failed to pull somanager:5000/security-onion-solutions/so-telegraf:2.3.91: Error 404: manifest for somanager:5000/security-onion-solutions/so-telegraf:2.3.91 not found: manifest unknown: manifest unknown
	Line 18292: Errors detected during setup; skipping post-setup steps to allow for analysis of failures.

Thanks,

Troy

**Comment by ghost** [comment]
I just tried a 3rd computer with the same result. Could this have something to do with the cpu? All three have 4 cores but they are old. I have newer sensors that are reinstalling just fine. 

The sosetup.log is the same as above. There are no entries in the errors.log and the telegraf log does not exist. 

Like I mentioned above. The first computer on which I reinstalled SO was running perfectly on .90. Nothing has changed since then, except that I had to rebuild the manager and the git clone gave me 91. I ran so-allow for all necessary IPs.

**Reply by dougburks** [reply]
> Could this have something to do with the cpu? All three have 4 cores but they are old. I have newer sensors that are reinstalling just fine.

What are the full specs of the machines?  Please check the Hardware Requirements:
https://docs.securityonion.net/en/2.3/hardware.html
